docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,79 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each candidate: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon participant registration. The CandID is not configurable.
```

Comment:
To follow up on Cecile's comment ^^  about consistency, 
change "candidate" to "candidate/participant" since "participant" is used more consistently below.
I find it reads funny to say "there are 2 IDs per candidate" followed by "this ID happens upon participant registration"

Reply:
well I'm only talking about the CandId in this section... theres a section for PSCID below

for the candidate/participant, since we use candidate in LORIS I'll switch them over to candidate for now

Reply:
done